### DISCUSSION TITLE
How to put this html file into GoldenDict-ng to become a dictionary?

### DISCUSSION THREAD

**Discussion by yshumy** [xiaoyifang/goldendict-ng#1593] [General]
我试了两个办法：
1.添加到网站：把文件的路径复制到网址里，可以打开，但是剪贴板内容不会自动进入搜索框执行搜索。
2.添加到程序：不知道怎么做，没成功。
html文件，见附件：
[部件檢索改代碼.html.zip](https://github.com/xiaoyifang/goldendict-ng/files/13771152/html.zip)

[a comment by an automated account is not shown]

**Comment by xiaoyifang** [comment]
添加本地的html，需要对html做些改造，你可以参考下这个。
https://github.com/goldendict/goldendict/issues/1680

[a comment by an automated account is not shown]

**Comment by yshumy** [comment]
> 添加本地的html，需要对html做些改造，你可以参考下这个。 [goldendict/goldendict#1680](https://github.com/goldendict/goldendict/issues/1680)

他这个不是本地html，而是网站。我的难题是怎样把热键取词的剪贴板内容放到本地html的搜索栏里，并自动执行搜索。麻烦帮忙看看我给的本地html的内容。

[a comment by an automated account is not shown]

**Comment by bAH7JgbXrMsXEhggm4** [comment]
> 添加本地的html，需要对html做些改造，你可以参考下这个。 [goldendict/goldendict#1680](https://github.com/goldendict/goldendict/issues/1680)

大佬，看到这个我感觉很巧，这不是我10天前在原版仓库提的issue嘛，想不到也引起了您的注意。那时候我还不知道有goldendict-ng版，今天改为使用ng版，发现原版发音的问题是不存在的 :blush:（我用的最新代码编译的）。
一切都很完美，左右边距，上边距都可以调节，只是有一个问题，高度无法调节,这导致信息显示不全，想看被遮挡的信息需要鼠标移过去滚轮翻页，已经提了 issue,您可以看下。

[a comment by an automated account is not shown]
